lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -1226,4 +1228,80 @@ sub get_fencing_mechanism {
     return ($supported_fencing_values{$fencing_type});
 }
 
+=head3 sdaf_upload_logs
```

Comment:
SDAF has been designed to interact to SUT like OnPrem tests thanks to console_redirection. Why not to call `ha_export_log` from `lib/hacluster.pm`. If it is not directly possible to call it, I suggest to adapt that one, maybe split it, and adapt both SDAF and OnPrem tests using it to share the code.

Reply:
Yes, I noticed and tried `ha_export_log` but it is not directly possible. So I decided to write a new one for sdaf.